database/db.init

```diff
@@ -0,0 +1,129 @@
+drop schema public cascade;
+create schema public;
+--DROP TABLE public."user";
+
+CREATE TABLE public."user"
+(
+    id       serial,
+    name     character varying(250),
+    email    character varying(250),
+    password character varying(250),
+    CONSTRAINT id PRIMARY KEY (id)
+) WITH (
+      OIDS = FALSE
+    );
+
+CREATE TABLE SUBJECT
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    user_id     SERIAL REFERENCES public."user" (id)
+);
+
+CREATE TABLE EXAM
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    subject_id  SERIAL REFERENCES SUBJECT (id),
+    pass_score  VARCHAR,
+    file_name   VARCHAR,
+    date        DATE,
+    user_id     SERIAL REFERENCES public."user" (id),
+    owners      VARCHAR
+);
+
+CREATE TABLE EXAM_OWNERS
+(
+    exam_id SERIAL REFERENCES EXAM (id),
+    user_id SERIAL REFERENCES public."user" (id)
+);
+
+CREATE TABLE EXAM_FILES
+(
+    id            SERIAL PRIMARY KEY UNIQUE,
+    file_name     VARCHAR,
+    file_path     VARCHAR,
+    students_work bool,
+    answers       bool,

```

Comment:
Seems redundant. If it's not students' work, it has to be answers. What else could it be?

Reply:
@MateuszKepczynski please have a look